com.photopea.Photopea.metainfo.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+    <id>com.photopea.Photopea</id>
```

Comment:
Since this no the photopea application but rather your webview wrapper then the id should be something else.

Reply:
What ID should it be? com.github.vikdevelop.photopea_app?

Reply:
yes